libp2p/transport/tcp/tcp.py

```diff
@@ -1,68 +1,74 @@
 import asyncio
+from socket import socket
+from typing import List
 
-import multiaddr
+from multiaddr import Multiaddr
 
 from libp2p.network.connection.raw_connection import RawConnection
+from libp2p.network.connection.raw_connection_interface import IRawConnection
+from libp2p.peer.id import ID
+from libp2p.transport.listener_interface import IListener
+from libp2p.transport.transport_interface import ITransport
+from libp2p.transport.typing import THandler
 
-from ..listener_interface import IListener
-from ..transport_interface import ITransport
+
+class TCPListener(IListener):
+    multiaddrs: List[Multiaddr]
+    server = None
+    handler = None
```

Comment:
Should it be
```suggestion
    handler: THandler
```
?

Reply:
same as https://github.com/libp2p/py-libp2p/pull/203#discussion_r310508651